nms/app/packages/magmalte/app/components/admin/AuditLog.js

```diff
@@ -13,157 +13,119 @@
  * @flow
  * @format
  */
-
-import Button from '@fbcnms/ui/components/design-system/Button';
+import ActionTable from '../ActionTable';
+import CardTitleRow from '../../components/layout/CardTitleRow';
 import DeviceStatusCircle from '@fbcnms/ui/components/icons/DeviceStatusCircle';
 import Dialog from '@material-ui/core/Dialog';
-import DialogActions from '@material-ui/core/DialogActions';
 import DialogContent from '@material-ui/core/DialogContent';
 import DialogTitle from '@material-ui/core/DialogTitle';
-import IconButton from '@material-ui/core/IconButton';
-import JSONTree from 'react-json-tree';
+import Grid from '@material-ui/core/Grid';
+import ListAltIcon from '@material-ui/icons/ListAlt';
 import LoadingFiller from '@fbcnms/ui/components/LoadingFiller';
-import MoreVert from '@material-ui/icons/MoreVert';
-import NestedRouteLink from '@fbcnms/ui/components/NestedRouteLink';
-import Paper from '@material-ui/core/Paper';
 import React from 'react';
-import Table from '@material-ui/core/Table';
-import TableBody from '@material-ui/core/TableBody';
-import TableCell from '@material-ui/core/TableCell';
-import TableHead from '@material-ui/core/TableHead';
-import TableRow from '@material-ui/core/TableRow';
-import Text from '../../theme/design-system/Text';
-import {colors} from '../../theme/default';
+import ReactJson from 'react-json-view';
 
-import {Route} from 'react-router-dom';
 import {makeStyles} from '@material-ui/styles';
 import {useAxios} from '@fbcnms/ui/hooks';
-import {useRouter} from '@fbcnms/ui/hooks';
+import {useState} from 'react';
 
 const useStyles = makeStyles(theme => ({
-  actionsCell: {
-    textAlign: 'right',
-  },
-  actionsColumn: {
-    width: '160px',
-  },
-  header: {
-    margin: '10px',
-    display: 'flex',
-    justifyContent: 'space-between',
-  },
-  paper: {
-    margin: theme.spacing(3),
-  },
-  iconButton: {
-    color: colors.primary.brightGray,
-    padding: '5px',
+  dashboardRoot: {
+    margin: theme.spacing(5),
   },
 }));
 
-const THEME = {
-  scheme: 'monokai',
-  base00: '#272822',
-  base01: '#383830',
-  base02: '#49483e',
-  base03: '#75715e',
-  base04: '#a59f85',
-  base05: '#f8f8f2',
-  base06: '#f5f4f1',
-  base07: '#f9f8f5',
-  base08: '#f92672',
-  base09: '#fd971f',
-  base0A: '#f4bf75',
-  base0B: '#a6e22e',
-  base0C: '#a1efe4',
-  base0D: '#66d9ef',
-  base0E: '#ae81ff',
-  base0F: '#cc6633',
+export type AuditLogRowType = {
+  id: string,
+  actingUserEmail: string,
+  mutationType: string,
+  objectType: string,
+  objectId: string,
+  mutationData: {},
 };
 
 export default function () {
```

Comment:
Could you add a name to this, eg. `AuditLog`, and another line at the bottom of the file:

```
export default AuditLog;
```

Reply:
Also a small docstring for the component would be helpful as well